components/kit/components/commerce/shopping/BookingCard.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { FC } from "react";
+
+const BookingCard: FC = () => {
+  return (
+    <div className="lg:flex  max-w-2xl mx-auto overflow-hidden rounded-lg m-24  shadow-lg">
```

Comment:
Can you implement the dark version of the component ? ( see : https://tailwindcss.com/docs/dark-mode)